source-reminder/requirements.txt

```diff
@@ -0,0 +1,5 @@
+gspread
```

Comment:
We've gotten minorly burned a couple times in situations where we don't pin a specific version, when we go to deploy something and suddenly there's a breaking change from an upgraded version that we didn't intend to do.  I think we could save our future selves some potential confusion by pinning the versions of these dependencies.  I think `pip freeze` _might_ be helpful for pinning versions.  We could be loose with the version requirements (but at least having it pinned to a package's major version I think would be helpful).